packages/gafl-webapp-service/src/handlers/__tests__/page-handler.spec.js

```diff
@@ -235,6 +237,35 @@ describe('The page handler function', () => {
       })
     )
   })
+
+  it.each([[PAYMENT_CANCELLED.uri], [PAYMENT_FAILED.uri], [AGREED.uri], [ORDER_COMPLETE.uri], [LICENCE_DETAILS.uri]])(
+    'sets notJourneyBeginning to true if not on licence_for or identify page',
+    async pageUri => {
+      const { get } = pageHandler('', 'view', '/next/page')
+      const toolkit = getMockToolkit()
+      const mockRequest = getMockRequest(null, pageUri)
+      await get(mockRequest, toolkit)
+      const pageData = toolkit.view.mock.calls[0][1]
+      expect(pageData).toEqual(
+        expect.objectContaining({
+          notJourneyBeginning: true
+        })
+      )
+    }
+  )
+
+  it.each([[IDENTIFY.uri], [LICENCE_FOR.uri]])('sets notJourneyBeginning to false if on licence_for or identify page', async pageUri => {
+    const { get } = pageHandler('', 'view', '/next/page')
+    const toolkit = getMockToolkit()
+    const mockRequest = getMockRequest(null, pageUri)
+    await get(mockRequest, toolkit)
+    const pageData = toolkit.view.mock.calls[0][1]
+    expect(pageData).toEqual(
+      expect.objectContaining({
+        notJourneyBeginning: false
+      })
+    )
+  })
```

Comment:
I think you've misunderstood me. We shoudn't have a value that we have to set on every page. If you reverse this to `journeyBeginning`, which gets set to true on the licence-for and renewal id pages, then in the layout.njk it can show the robots meta tag when that's not true. What you're doing, whilst it will work, means setting the value on every single page other than the two I've just mentioned - obviously far more onerous.